lib/dot/ref.jst

```diff
@@ -58,8 +58,9 @@
 
 {{? $refCode }}
   {{# def.beginDefOut}}
-    {{# def._validateRef:$refCode }}
+    {{# def._validateRef }}
   {{# def.storeDefOut:__callValidate }}
+  var $refCodeFn = {{= $refCode }};
```

Comment:
I don't understand these changes. It looks like `$refCodeFn` is assigned after it is used.